src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,60 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import lotto.controller.generator.LottoGenerator;
+
+public class Lotto {
+
+    public static final int LOTTO_POSSESSION_NUMBER = 6;
+    public static final double BONUS_MATCHING_COUNT = 5.5;
+    private final List<Integer> numbers;
+
+    public Lotto(LottoGenerator lottoGenerator) {
+        numbers = new ArrayList<>(lottoGenerator.generateNumbers());
+    }
+
+    public Lotto(List<Integer> selectedNumber) {
+        numbers = new ArrayList<>(selectedNumber);
+    }
```

Comment:
```suggestion
    public Lotto(LottoGenerator lottoGenerator) {
        this(lottoGenerator.generateNumbers());
    }

    public Lotto(List<Integer> selectedNumber) {
        numbers = new ArrayList<>(selectedNumber);
    }
```

인스턴스 변수의 초기화는 하나의 생성자에서 해주는 것이 좋습니다.
초기화 또한 중복이라는 관점에서 어떠한 차이가 있을지 고민해보아요 :)

Reply:
넵! 앞의 도메인 부분 수정하면서 같이 수정하였습니다!